static/app/test/specs/entities.spec.js

```diff
@@ -8,27 +8,102 @@ var Radio = require('backbone.radio');
 
 // App
 var EntitiesAPI = require('app/entities').API;
+var Move = require('app/entities').Move;
 var channel = Radio.channel('global');
 
 describe('Entities', function(){
     describe('API', function(){
         describe('initialization', function(){
-            it('registers an entities:spots request handler');
-            it('registers an entities:moves request handler');
-            it('registers an entities:move request handler');
-            it('registers an entities:move:reset request handler');
+            
+            it('registers an entities:spots request handler', function(){
+                var entitiesAPI = new EntitiesAPI();
+                var spots = "something";
+                this.replySpotsStub = sinon.stub(EntitiesAPI.prototype, 'replySpots', function(){ return spots });
```

Comment:
In general, it's better practice to stub out the method on the instance not the prototype, e.g.:

```javascript
sinon.stub(entitiesAPI, 'replySpots', function(){ ...
```
When we run the tests, it uses the same global context throughout. This means altering the prototype can be potentially dangerous. You're doing the right thing here and making sure to restore the method at the end of the test. But if you either forget to restore the method after the test is finished or if the restore does not happen because the test errors or fails before it gets to it, it can cause other tests to fail, which can be annoying to debug.

All that said, sometimes it is necessary to stub out the prototype because you don't have access to the actual instance. But this isn't one of those cases.